README.md

```diff
@@ -31,7 +31,15 @@ Here are the commands you can run. Click on each one for detailed documentation
 - [`docs`](https://extensionworkshop.com/documentation/develop/web-ext-command-reference#web-ext-docs)
   - Open the `web-ext` documentation in a browser
 
-## Installation from npm
+## Installation
+
+### Using Homebrew
```

Comment:
Please move that section right before "Installation from source" and add:

```suggestion
### Using Homebrew (unofficial)

The community maintains a `web-ext` formula.
```

Reply:
Done.